### PR TITLE
Add optical flow doc from devguide

### PR DESCRIPTION
**Pull request by hamishwillee**
This merges optical flow doc from devguide. The new doc is a parent of PX4 Flow. I moved the duplicate PX4 flow info INTO the sub document. This is all now a lot more logical.

As last commit I also added a note about electromagnetic interference being an issue to address https://github.com/PX4/Devguide/issues/562

When this is merged, need to also merge: https://github.com/PX4/Devguide/pull/563

### CODE REVIEW

File: en/sensor/optical_flow.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Optical Flow uses a downward facing camera and a downward facing distance sensor for position estimation. 
+Optical Flow based navigation is supported by: EKF2 and LPE.
+
+## Setup
+
+An Optical Flow setup requires a downward facing camera which publishes to the [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD) topic and a [distance sensor](../sensor/rangefinders.md) (preferably a LiDAR) publishing messages to the [DISANCE_SENSOR](https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR) topic.
+
+The output of the flow has to be as follows
+
+| Vehicle movement | Integrated flow |
```
Comment (hamishwillee) [inline en/sensor/optical_flow.md]: I did not know what this meant - ie @bkueng Is this something that someone has to do? Where is the X and Y set - is it in the OPTICAL_FLOW_RAD?

File: en/sensor/optical_flow.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+| -- | -- |
+| Forwards | + Y |
+| Backwards | - Y |
+| Right | - X |
+| Left | + X |
+
+For pure rotations the `integraded_xgyro` and `integraded_x` (respectively `integraded_ygyro` and `integraded_y`) have to be the same.
```
Comment (hamishwillee) [inline en/sensor/optical_flow.md]: What are integraded_xgyro and integraded_x? 
Also, are these miss-spelled or do they reflect code?

File: en/sensor/px4flow.md [hunk 3]
```diff
@@ -44,36 +46,52 @@ If out of stock the software-compatible, but not connector-compatible version ca
...
 ![PX4Flow align with Pixhawk](../../assets/hardware/sensors/px4flow/px4flowalignwithpixhawk.jpg)
 
+On **PX4**, the orientation should be set using the parameter [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT). 
+The above orientation maps to `SENS_FLOW_ROT`=270 degrees (the default). 
 
-The orientation of PX4Flow can be set using the parameter [SENS_FLOW_ROT](https://docs.px4.io/en/advanced_config/parameter_reference.html#SENS_FLOW_ROT)
+Make sure the the PX4Flow board is well dampened.
 
-> **Warning** This parameter can only be applied to PX4 flight stack and is **not** part of the sensor itself settings
+> **Warning** PX4Flow emits a significant amount of electromagnetic radiation, and should be placed as far away from other electronics (in particular GPS modules) as possible.
```
Comment (hamishwillee) [inline en/sensor/px4flow.md]: This is note about radiation - see https://github.com/PX4/Devguide/issues/562

File: en/sensor/px4flow.md [hunk 4]
```diff
@@ -82,7 +100,36 @@ Address | Bit 2 | Bit 1 | Bit 0
...
+If you modify the I2C address of the PX4Flow, make sure to start the PX4 driver with the correct address:
+```
+px4flow start                  # address=0x42 (default)
+px4flow stop
+px4flow start -a 0x45          # address=0x45
+```
+
+
+## Focusing the Lens
+
```
Comment (hamishwillee) [inline en/sensor/px4flow.md]: Was not sure this is strictly necessary - it was in the devguide. However below somewhere we mention that the optics arrive focussed. I suspect this is useful though?

File: en/sensor/optical_flow.md (hunk 2 above)
Comment (hamishwillee) [reply en/sensor/optical_flow.md]: Yes, but I mean are these miss-spelled here? Ie it is `integraDed_xgyro` above, but integraTed_xgyro` in [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD)?

File: en/sensor/optical_flow.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+# Optical Flow
+
+Optical Flow uses a downward facing camera and a downward facing distance sensor for position estimation. 
+Optical Flow based navigation is supported by: EKF2 and LPE.
+
+## Setup
+
+An Optical Flow setup requires a downward facing camera which publishes to the [OPTICAL_FLOW_RAD](https://mavlink.io/en/messages/common.html#OPTICAL_FLOW_RAD) topic and a [distance sensor](../sensor/rangefinders.md) (preferably a LiDAR) publishing messages to the [DISANCE_SENSOR](https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR) topic.
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/sensor/optical_flow.md]: OK, so I have separated out info about the mavlink topics in https://github.com/PX4/px4_user_guide/pull/300/commits/ade2d378f797a654a59960b00c8aa4d943e27fdd

Can you check this bit?

File: en/sensor/optical_flow.md (hunk 2 above)
Comment (bkueng) [reply en/sensor/optical_flow.md]: Ah, right. Yes they are simply miss-spelled.

**Pull request merged**